providers/src/airflow/providers/standard/operators/python.py

```diff
@@ -417,14 +393,15 @@ class _BasePythonVirtualenvOperator(PythonOperator, metaclass=ABCMeta):
         "prev_start_date_success",
         "prev_end_date_success",
     }
+
     AIRFLOW_SERIALIZABLE_CONTEXT_KEYS = {
         "macros",
         "conf",
         "dag",
         "dag_run",
         "task",
         "params",
-        "triggering_asset_events",
+        "triggering_asset_events" if AIRFLOW_V_3_0_PLUS else "triggering_dataset_events",
```

Comment:
The main branch already includes asset event changes compatible with Airflow 3. Since the standard provider needs to support Airflow 2.10, which includes the dataset event feature, I’ve updated this to ensure compatibility with both versions.

Reply:
Yeah. When Python operator wa part of "Airflow" it was never supposed to be run against old airflow version, but when we move to provider, we also run it against previous airflow versions :)